test/common/algos.mocha.coffee

```diff
@@ -185,7 +185,42 @@ describe 'User', ->
     expect(user.stats.buffs.str).to.be 1
     expect(user.achievements.perfect).to.be 2
 
+  describe 'Resting in the Inn', ->
+    user = null
+    cron = null
+    beforeEach ->
+      user = newUser()
+      user.preferences.sleep = true
+      cron = -> user.lastCron = moment().subtract(1, 'days');user.fns.cron()
+      user.dailys = []
+      _.times 2, -> user.dailys.push shared.taskDefaults({type:'daily'})
+    it "doesn't change on cron", ->
```

Comment:
I'd be more explicit about what doesn't change, maybe `it "remains in the inn on cron", ->`